test/test_p4est_2d.jl

```diff
@@ -655,6 +655,37 @@ end
     @test isapprox(lift, 0.029076443678087403, atol = 1e-13)
     @test isapprox(drag, 0.13564720009197903, atol = 1e-13)
 end
+
+@trixi_testset "elixir_euler_blast_wave_pure_fv.jl" begin
+    @test_trixi_include(joinpath(pkgdir(Trixi, "examples", "tree_2d_dgsem"),
+                                 "elixir_euler_blast_wave_pure_fv.jl"),
+                        l2=[
+                            0.39957047631960346,
+                            0.21006912294983154,
+                            0.21006903549932,
+                            0.6280328163981136,
+                        ],
+                        linf=[
+                            2.20417889887697,
+                            1.5487238480003327,
+                            1.5486788679247812,
+                            2.4656795949035857,
+                        ],
+                        tspan=(0.0, 0.5),
+                        # Let this test run longer to cover some lines in flux_hllc
+                        coverage_override=(maxiters = 10^5, tspan = (0.0, 0.1)),
```

Comment:
Required?

Reply:
Have not checked, just copied this from https://github.com/trixi-framework/Trixi.jl/blob/91eac3ce395a107158b9a1ed9cbe213dd5606ee1/test/test_tree_2d_euler.jl#L311

Reply:
Please remove it - it should not be needed here as the comment explains